CadRevealFbxProvider/FbxMeshWrapper.cs

```diff
@@ -32,11 +31,11 @@ public static IntPtr GetMeshGeometryPtr(FbxNode node)
 
     // the underlying umanaged code allocates memory, you must call mesh_clean_memory to free it later
     [DllImport(FbxLib, CallingConvention = CallingConvention.Cdecl, EntryPoint = "mesh_get_geometry_data")]
-    private static extern IntPtr mesh_get_geometry_data(IntPtr mesh, bool ignore_normals); // IntPtr out is FbxMesh*
+    private static extern IntPtr mesh_get_geometry_data(IntPtr mesh); // IntPtr out is FbxMesh*
 
     public static Mesh? GetGeometricData(IntPtr meshPtr)
     {
-        var geomPtr = mesh_get_geometry_data(meshPtr, true); // Reveal does not use surface normals, hence we ignore them and treat vertices with different normals, but same location, as equivalent.
+        var geomPtr = mesh_get_geometry_data(meshPtr); // Reveal does not use surface normals, hence we ignore them and treat vertices with different normals, but same location, as equivalent.
```

Comment:
Update or move the comment closer to the place where we ignore normals

Reply:
Maybe just remove the comment?